static/app/views/nav/primary/components.tsx

```diff
@@ -313,10 +459,28 @@ export const NavButton = styled('button', {
   ${baseNavItemStyles}
 `;
 
-export const SidebarItemUnreadIndicator = styled('span')`
+interface NavButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  isMobile: boolean;
+}
+
+export const NavButton = styled((p: NavButtonProps) => {
+  const theme = useTheme();
+  if (theme.isChonk) {
+    return (
+      <ChonkNavButton
+        {...p}
+        aria-label={p['aria-label'] ?? ''}
+        size={p.isMobile ? 'zero' : undefined}
+      />
+    );
+  }
+  return <StyledNavButton {...p} />;
+})``;
+
+export const SidebarItemUnreadIndicator = styled('span')<{isMobile: boolean}>`
```

Comment:
@JonasBa I noticed one difference - this dot is now a bit too low.

Before/after:

![CleanShot 2025-04-09 at 08 55 23@2x](https://github.com/user-attachments/assets/99d50d8a-92ba-4406-9c05-ce762ff656ab)
![CleanShot 2025-04-09 at 08 55 27@2x](https://github.com/user-attachments/assets/f6b51d7c-c229-474e-a2c1-4b653d35d765)

Reply:
Ah, good catch! Let me move it higher up!

Reply:
Fixed upstream!